app/components/mandatees/mandatees-domains-panel/mandatees-domains-panel-edit.hbs

```diff
@@ -53,9 +53,10 @@
                 </td>
               </Mandatees::DomainsFieldsForMandatee>
               <td data-test-mandatee-panel-edit-row-submitter>
-                <WebComponents::VlToggle
-                  @value={{eq mandatee this.submitterBuffer}}
-                  @valueChanged={{fn this.toggleIsSubmitter mandatee}}
+                <Auk::Toggle
+                  id="item-show-in-newsletter"
```

Comment:
deze id klopt niet, het gaat hier over welke minister submitter is

Reply:
geen label, dus ook geen id nodig

Reply:
Vermoedelijk gekopieerd vanuit een ander component en vergeten te verwijderen. Goed opgemerkt!